cocos/network/WebSocket.cpp

```diff
@@ -646,6 +648,7 @@ void WebSocket::send(const std::string& message)
         // In main thread
         Data* data = new (std::nothrow) Data();
         data->bytes = (char*)malloc(message.length() + 1);
+        assert(data->bytes);
```

Comment:
Why to assert here? Does it possible return nullptr?

Reply:
yes if not enough memory and it is to make static analyser happy